frontend/src/styles.css

```
@@ -1147,3 +1147,31 @@ button:disabled:hover,
   transform: none;
   border-color: var(--palette-borders-separators);
 }
+
+.form-input,
+.form-textarea,
+.form-select {
+  background-color: var(--palette-accent-white) !important;
+  color: var(--palette-primary-text) !important;
+  border-color: var(--palette-borders-separators) !important;
+}
+
+input::placeholder,
+textarea::placeholder {
+  color: var(--palette-secondary-text);
+}
+
+@media (prefers-color-scheme: dark) {
+  input[type='text'],
+  input[type='email'],
+  input[type='password'],
+  textarea,
+  select,
+  .form-input,
+  .form-textarea,
+  .form-select {
+    background-color: var(--palette-accent-white) !important;
+    color: var(--palette-primary-text) !important;
+    border-color: var(--palette-borders-separators) !important;
+  }
```

Forcing the color values used by DaisyUI might work - it looks like it should also apply to the login form that Lore was working on.
```suggestion
   :root {
    --color-base-content: oklch(21% 0.006 285.885);
    --base-color-100: oklch(100% 0 0);
  }
  
  .bg-base-100,
  .input,
  .modal-box {
    background-color: var(--palette-accent-white);
  }
```